db/changelog/v-2/02-changeset-test.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file can be named `02-changeset-execution`, why test?

Reply:
Forgot to rename